postgres/assets/configuration/spec.yaml

```diff
@@ -320,6 +320,22 @@ files:
           value:
             type: integer
             example: 10000
+    - name: query_settings
```

Comment:
Do we need this to be configurable? 

If the settings are not dynamic (meaning they require a restart to enable), we really only need to query for them once when the check starts, meaning we don't need a configurable interval. We also likely don't need the ability to enable/disable this collection either.